easybuild/easyconfigs/h/HIP/hipamd-rocm-4.5.0-remove-testing.patch

```diff
@@ -0,0 +1,34 @@
+--- hipamd-rocm-4.5.0/CMakeLists.txt.orig	2021-11-12 14:26:59.514400918 +0100
```

Comment:
@nordmoen This patch file is still missing a description, can you clarify why this stuff needs to be stripped out?

Reply:
fixed in 38fa5c1